source/codegen/templates/proto_helpers.mako

```diff
@@ -100,7 +100,18 @@ message ${common_helpers.snake_to_pascal(function)}Request {
 %>\
 message ${common_helpers.snake_to_pascal(function)}Response {
 % for parameter in response_parameters:
-  ${parameter["type"]} ${parameter["name"]} = ${parameter["grpc_field_number"]};
+<%
+  parameter_name = parameter["name"]
+%>\
+% if (parameter.get("is_get_last_error_output_param", True)):
+%   if ((parameter_name == "error_message") | (parameter_name == "error_num")):
```

Comment:
We shouldn't need this check on if the param name is "error_message" or "error_num". I think you just need to modify the first statement to default to False instead if the `is_get_last_error_output_param` isn't found in the parameter dictionary:

So I think it'll produce the same generated output if it's just a single if/else here with `paramater.get("is_get_last_error_output_param", False)`

Reply:
Yeah, and for the record. `parameter_name in ["error_message", "error_num"]` is preferred in python. And (logical) `or` instead of (bitwise) `|`.